tests/tcp_wrk.ts

```diff
@@ -0,0 +1,22 @@
+// Used for benchmarking Deno's networking. See tools/wrk_benchmark.py
+// TODO Replace this with a real HTTP server once
+// https://github.com/denoland/deno/issues/726 is completed.
+import * as deno from "deno";
+let addr = deno.args[1] || "127.0.0.1:4500";
+let listener = deno.listen("tcp", addr);
+const response = new TextEncoder().encode(
+  "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n"
+);
+
+async function main() {
+  console.log("Listening on", addr);
+  while (true) {
+    const conn = await listener.accept();
```

Comment:
this is not keep-alive, the snippet I posted is

Reply:
Let's do simple requests for now, since this thing isn't even actually parsing HTTP. I've changed the Node program to "Connection: close"

Reply:
Ugh... you want to measure how fast the operating system can create sockets or what? That's not #977 but more of a "connection benchmark" like #899 only not really..

Reply:
That's clearly not at all what it's measuring, since it's much slower than Node. Due to how all of our "syscalls" (AKA "ops") go through a central infrastructure, I'm rather confident that Deno would be fast at a keep alive benchmark if and only if it was fast here. We'll get to the keep alive benchmark - but I'd rather do that when we're measuring an actual HTTP stack - since keep-alive has complex semantics. We have bottlenecks to address which are visible with this benchmark.

Reply:
Keep-alive is not any more complex or hacky than what you already have now, I wrote you a fully working wrk-compatible keep-alive snippet ready to be used as is. I think post 1997 is a better target than pre. Esp. with HTTP2 being essentially only keep-alive.

Reply:
in any case, this is better than nothing